v5/core/utils.go

```diff
@@ -255,3 +256,18 @@ func SliceContains(slice []string, contains string) bool {
 	}
 	return false
 }
+
```

Comment:
For reference, the platform services Go SDK has a few utility methods scattered among the integration tests:
* https://github.com/IBM/platform-services-go-sdk/blob/74f8fd8344851feceeb0a5ebc09b10d0a16c67cb/usermanagementv1/user_management_v1_integration_test.go#L306
* https://github.com/IBM/platform-services-go-sdk/blob/74f8fd8344851feceeb0a5ebc09b10d0a16c67cb/iamidentityv1/iam_identity_v1_integration_test.go#L583
* https://github.com/IBM/platform-services-go-sdk/blob/74f8fd8344851feceeb0a5ebc09b10d0a16c67cb/enterpriseusagereportsv1/enterprise_usage_reports_v1_integration_test.go#L240
They all pretty much work the same way.  We can replace those with the new core function when it's available